composer.json

```diff
@@ -14,7 +14,8 @@
         }
     },
     "require": {
-        "amphp/artax": "^3",
+        "php": "^7.0",
```

Comment:
Yeah. That is why I suggested `^7.0` because `7.0.* || 7.1.*` is overkill and `>=7.0` is too generous.